### PR TITLE
Share mev gas price and gas used in hints

### PR DESCRIPTION
**Pull request by dvush**
Currently, it's deployed on goerli for testing.

### CODE REVIEW

File: specs/events/v0.1.md [hunk 1]
```diff
@@ -44,6 +48,16 @@ Transactions from the event. If the event itself is a transaction, txs will only
 | `functionSelector` | Hex-string | 4-byte function selector. |
 | `callData` | Hex-string | Calldata of the transaction. |
 
+### `mevGasPrice`
+
+Hex-string of the MEV gas price paid by the transaction rounded up to the 3 most significant hex digits.
+
+MEV gas price is calculated as coinbase delta after inserting the bundle divided by the gas used.
```
Comment (epheph) [inline specs/events/v0.1.md:55]: as mentioned in slack, would be great to have the formula here.

File: specs/events/v0.1.md [hunk 2]
```diff
@@ -44,6 +48,16 @@ Transactions from the event. If the event itself is a transaction, txs will only
...
+
+### `gasUsed`
+
+Hex-string of the gas used by the transaction rounded up to the 3 most significant hex digits.
```
Comment (epheph) [inline specs/events/v0.1.md:59]: Do we have a discussion where we consider the implication of this? I think we could detect things like add liquidity vs swap....

It also makes me think of what happens when we have a uni v3 market with a ton of liquidity in the nearest tick on the sell side, but spread out over a significant number of ticks on the buy side... could we determine which direction a swap is occurring by looking at gasUsed?

**Review by epheph — APPROVED** [review]
Left a couple concerns. would be interested to talk about gasUsed more.

**Comment by dvush** [comment]
Close because its will not be deployed to mainnet in this form

**Pull request closed without merging**